app/src/main/java/com/example/tictactoe/dialogs/MatchEndDialog.kt

```diff
@@ -1,11 +1,12 @@
-package com.example.tictactoe
+package com.example.tictactoe.dialogs
 
 import android.app.AlertDialog
 import android.content.Context
 import android.view.LayoutInflater
 import android.view.View
 import android.widget.Button
 import android.widget.TextView
+import com.example.tictactoe.R
```

Comment:
Why is new import required?

Reply:
Previously, it was in the main package and is now moved to another package, so this import is required.